DeviceDetector.php

```diff
@@ -1070,10 +1070,17 @@ protected function parseDevice(): void
         /**
          * All devices that contain Andr0id in string are assumed to be a tv
          */
-        if ($this->matchUserAgent('Andr0id|(?:Android(?: UHD)?|(?<!Xming )Google) TV|\(lite\) TV|BRAVIA| TV$')) {
+        if ($this->matchUserAgent('Andr0id|(?:Android(?: UHD)?|Google) TV|\(lite\) TV|BRAVIA| TV$')) {
             $this->device = AbstractDeviceParser::DEVICE_TYPE_TV;
         }
 
+        /**
+         * All devices that contain these fragments are assumed to be a peripheral
+         */
+        if ($this->matchUserAgent('STI6110|Xming')) {
+            $this->device = AbstractDeviceParser::DEVICE_TYPE_PERIPHERAL;
+        }
+
```

Comment:
I think the best option would be, add device type check.
Something like:
```
      /**
         * All devices that contain Andr0id in string are assumed to be a tv
         */
        $hasDeviceTvType = false === \in_array($this->device, [
            AbstractDeviceParser::DEVICE_TYPE_TV,
            AbstractDeviceParser::DEVICE_TYPE_PERIPHERAL,
        ]) && $this->matchUserAgent('Andr0id|(?:Android(?: UHD)?|Google) TV|\(lite\) TV|BRAVIA| TV$');

        if ($hasDeviceTvType) {
            $this->device = AbstractDeviceParser::DEVICE_TYPE_TV;
        }
```
(If we have previously determined the device type DEVICE_TYPE_TV or DEVICE_TYPE_PERIPHERAL based on the device type definition, this condition does not need to be checked.)